src/main/java/uk/gov/hmcts/darts/audio/entity/AudioRequest.java

```diff
@@ -23,26 +23,27 @@ public class AudioRequest {
 
     public static final String REQUEST_ID = "request_id";
     public static final String CASE_ID = "case_id";
-    public static final String EMAIL_ADDRESS = "email_address";
+    public static final String REQUESTER = "requester";
     public static final String START_TIME = "start_time";
     public static final String END_TIME = "end_time";
     public static final String REQUEST_TYPE = "request_type";
     public static final String STATUS = "status";
     public static final String ATTEMPTS = "attempts";
+    public static final String OUTBOUND_LOCATION = "outbound_location";
     public static final String CREATED_DATE_TIME = "created_date_time";
     public static final String LAST_UPDATED_DATE_TIME = "last_updated_date_time";
     public static final String TABLE_NAME = "audio_request";
 
     @Id
     @Column(name = REQUEST_ID)
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
-    private Integer id;
+    private Integer requestId;
 
     @Column(name = CASE_ID)
     private String caseId;
 
-    @Column(name = EMAIL_ADDRESS)
-    private String emailAddress;
+    @Column(name = REQUESTER)
+    private String requester;
 
     @Column(name = START_TIME)
     private Timestamp startTime;
```

Comment:
Yeh, good point. This should be consistent throughout - is ZonedDateTime the one most commonly used?

Reply:
I used time timestamp because that what was defined in the ticket. Also, at time of writing the entity, there were other examples such as the Notification entity.

Reply:
@jackmaloney No it's up for discussion, the entities at the moment are using the deprecated Date / Timestamp types.
I'm expecting the database to store the date/time as UTC globally. Then the front end to adjust for `London` BST time?

Reply:
Have created this ticket to cover investigation into this https://tools.hmcts.net/jira/browse/DMP-357 
Suggest we get this PR through as it is, and refactor in the future off the back of that spike.